### PR TITLE
Portfolio: Add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by jcmunav63**
**Description**
For the third milestone of your portfolio website, you will:
- Add a contact form to your portfolio and the portfolio of your coding partner.
- Make sure that you guide your users' experience in a way that helps them fill in the form without frustration.

**General requirements**
- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- Make sure that you used the correct [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
- Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

**HTML/CSS requirements**
- Follow our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

**Project requirements**
- The section you need to build for this milestone is:
   - The contact form.
   - And the footer (if present in the design).

We included a screenshot of Template 1 for reference, but you should follow the template that you chose.
- Make sure that:
   - Name field accepts up to 30 chars.
   - Email field accepts only emails.
   - Message field accepts up to 500 chars.
   - All fields have placeholder text.
   - All fields have the correct type.
   - A user cannot submit the form if one or more of the fields is empty.
- Collect data by using the [Formspree service](https://formspree.io/html).
- In order to lay out the elements on the page you should use Flexbox.
- You don't need to implement any functionality that requires JavaScript.
- You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- Implement the button interactions (enable, hover, etc.).
- The website should be responsive, following these breakpoints:
   - Minimum mobile screen size should be 375px.
   - Minimum desktop screen size should be 768px.
- Use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels.

**Work and submission mode**
- You should implement the above requirements in all repositories in your pair-programming group. That means that each group member needs to open a new pull request in their repo.
- We will check the commit history to make sure that everybody has contributed code on each project.
- You should ask for a review and submit this activity individually.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,192 +1,480 @@
...
+                src="images/right-arrow.png"
+              />
+            </button>
+          </div>
+          <hr />
+        </li>
+      </ul>
+    </section>
+
+    <footer class="footer">
```
Comment (stanleySimeon) [inline index.html:457]: While you made great effort on this block, but there are a few things you have to fix before going to the next steps.

- Kindly, I suggest you remove the `section` tag inside the `footer` tag for good semantics and best practices. 😄

File: index.html [hunk 2]
```diff
@@ -1,192 +1,480 @@
...
+          </p>
+          <div class="box2">
+            <input id="name" name="name" type="text" class="poppins" placeholder="Lorem Ipsum" maxlength="30" required>
+          </div>
+          <div class="box2">
+            <input id="email" name="email" type="email" class="poppins" placeholder="loremipsum@gmail.com" required>
+          </div>
+          <div class="box3">
+            <textarea id="message" name="message" class="poppins" rows="5" cols="43" placeholder="Write your message here..." maxlength="500" required></textarea>
+          </div>
```
Comment (stanleySimeon) [inline index.html:474]: Kindly, I suggest you remove all this `div` tags in replace them by `ul` and `li` to group them.

It seems you forgot to close the `input` tags element, kindly, I suggest you close the `input` as follow: `<input ... />` for good semantics and best practices. 😄

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hi @jcmunav63, @mwovi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Correct use of Gitflow ✔️
- Linter checks are passed ✔️
- Good PR description ✔️
- Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- adding your live demo link

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @jcmunav63, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] Good UI design :white_check_mark: 
- [x] Contact form works as expected :white_check_mark: 
- [x] GitHub flow is used correctly :white_check_mark: 
- [x] Implemented all the required changes from the previous reviewer :white_check_mark:  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**